env0/data_environment.go

```diff
@@ -98,6 +98,25 @@ func dataEnvironment() *schema.Resource {
 				Description: "The token id used for repo integrations (Used by Gitlab or Azure DevOps)",
 				Computed:    true,
 			},
+			"sub_environment_configuration": {
+				Type:        schema.TypeList,
+				Description: "the sub environments of the workflow environment. (Empty for non workflow environments)",
+				Computed:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:        schema.TypeString,
```

Comment:
for now I have added 'alias' and 'id'. More can be added in the future if required.